TestResultSummaryService/routes/postTapFiles.js

```diff
@@ -11,14 +13,19 @@ const Parsers = require(`../parsers/`);
  */
 
 module.exports = async (req, res) => {
-    //const zip = req.query.url;
-    const zip = __dirname;
+    const url = req.query.url;
     const tapParser = Parsers['Tap'];
 
-    if (tapParser.canParse(zip) == false) {
-        res.send({ error: `invalid zipfile: ${zip}` });
+    if (tapParser.canParse(url) == false) {
+        res.send({ error: `invalid zipfile: ${url}` });
     } else {
-        const status = await tapParser.parse(zip);
+        const { data } = await axios.get(url, {
+            responseType: 'arraybuffer',
+        });
```

Comment:
Instead of introducing a new fetch library - `axios`, could you use the existing one - `got`? You may need to check the return data. It may be `response.body`.

```
const response = await got.get(this.url).buffer()

```

https://github.com/sindresorhus/got#readme

Reply:
Also, we need to add try catch around this code, just in case anything invalid is provided.